src/EnergyPlus/DataSurfaces.cc

```diff
@@ -86,6 +94,14 @@ namespace DataSurfaces {
 	using namespace DataPrecisionGlobals;
 	using namespace DataVectorTypes;
 	using namespace DataBSDFWindow;
+  using namespace DataHeatBalance;
```

Comment:
The style guidelines do probably need to be updated. Big things are to be consistent, use tabs, have one space between parentheses and other control mechanisms, try to avoid excessive `using namespace Foo;`, and try to declare variables close to their usage.